app/views/coordinators/_form.html.erb

```diff
@@ -1,18 +1,44 @@
-<%= simple_form_for(@coordinator) do |f| %>
-  <%= f.error_notification %>
+<% if @coordinator.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= pluralize(@coordinator.errors.count, "error") %> prohibited
+      this coordinator from being saved:
+    </h2>
+    <ul>
+    <% @coordinator.errors.full_messages.each do |msg| %>
+      <li><%= msg %></li>
+    <% end %>
+    </ul>
+  </div>
+<% end %>
 
-  <div class="form-inputs">
-    <%= f.input :name %>
-    <%= f.input :date_of_birth, input_html: { class: 'datepicker' }, placeholder: "YYYY-MM-DD", as: :string, required: false %>
-    <%= f.input :email, as: :string, label: "email", maxlength: 255, required: false, input_html: { class: "mask_email" } %>
-    <%= f.input :phone_number, maxlength: 14, placeholder: "(###) ###-####", required: false, input_html: { class: "mask_phone" } %>
+<form role="form">
 
-  </div>
+  <section id="contactInfo">
+    <h4>Information</h4>
+    <hr>
+    <div class="row">
```

Comment:
Each row can hold a total of 12 column units. Here you have a row holding 4+4 and then another row with 4+4. Let's try to fill up the whole row. You can either remove the second row and change the col sizes to 3 for all of them, or you can keep the second row but make the col sizes all 6 so each row adds up to 6. If you have any other ideas feel free to go ahead with them.